sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -186,6 +186,7 @@ def from_estimator(
         *,
         sample_weight=None,
         pos_label=None,
+        drop_intermediate=True,
```

Comment:
I don't think that we can use `True` as a default here. We will need to be backward compatible.

We would need to make it `False`, by default. Then we can decide whether or not this is worth changing the default and launching a deprecation cycle.